source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -54,6 +54,8 @@ namespace AdaptiveCards
                 String OverflowButtonText;
                 String OverflowButtonAccessibilityText;
 
+                Boolean AddSelectActionMargin;
```

Comment:
General note: I tried looking for other ways we could override this behavior (custom styling to override the onHover foreground and background colors?), but didn't see anything. Open to any ideas here :)

Reply:
Resolving reason: I investigated using custom styling instead, but this does not work because programmatically set styles take precedence over resource dictionaries.